repl/src/test/scala/org/apache/spark/repl/ReplSuite.scala

```diff
@@ -86,31 +84,6 @@ class ReplSuite extends SparkFunSuite with BeforeAndAfterAll {
       "Interpreter output contained '" + message + "':\n" + output)
   }
 
-  test("propagation of local properties") {
```

Comment:
Is this test just obsolete now? Just want to understand where we might be taking away tests.

Reply:
I moved it to [Repl2Suite](https://github.com/apache/spark/pull/28545/files#diff-d5d7e0ac42c11024a7e6f6c9e346bdb1R34) (because it's different between Scala 2.12 and 2.13)